docs/conf.py

```diff
@@ -112,3 +112,6 @@ def autodoc_mutators_skip_member(app, what, name, obj, skip, options):
 
 def setup(app):
     app.connect('autodoc-skip-member', autodoc_mutators_skip_member)
+
+# Allow to use "ddsmt" in "command-output" directives.
+os.environ['PATH'] = os.path.join(__root_dir, "bin") + os.pathsep + os.environ['PATH']
```

Comment:
We use `sys.path.insert(0, ...)` in other places, I think it's the cleaner way.
```suggestion
sys.path.insert(0, os.path.join(__root_dir, "bin"))
```

Reply:
That won't work because `sys.path` is the Python module path, we need (the correct version of) `ddsmt` in `$PATH` for things like `.. command-output:: ddsmt --help-all`.
Maybe there is a more Pythonic way to do this; I'm not sure.

Reply:
Hm, interesting. Good to know.